homework-submission/src/Components/list.css

```diff
@@ -0,0 +1,104 @@
+@keyframes rainbow {
+  0% {
+    color: rgb(209, 23, 23);
+  }
+  25% {
+    color: rgb(157, 17, 192);
+  }
+  50% {
+    color: rgb(11, 66, 155);
+  }
+  75% {
+    color: rgb(9, 110, 56);
+  }
+  100% {
+    color: rgb(179, 22, 17);
+  }
+}
+
+.no-todos {
+  font-size: 24px;
+  font-weight: 700;
+  min-width: 200px;
+  padding: 75px 0;
+  text-align: center;
+  margin: 0 auto;
+  color: #222;
+  animation: rainbow 10s infinite;
+  transform: scale(1.2);
+}
+
+.list {
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+  width: 600px;
+  max-height: 70vh;
+  overflow-y: scroll;
+  margin: 50px auto;
+  padding: 50px 80px 70px 60px;
+  border-radius: 15px;
+  box-shadow: 0 0 50px 3px rgba(0, 0, 0, 0.075),
+    0 0 3px 2px rgba(51, 51, 51) inset;
+  background: linear-gradient(
+    to bottom,
+    rgb(224, 224, 224),
+    rgba(15, 15, 15, 0.151)
+  );
+  margin-top: 25px;
+}
+
+.list li {
+  padding: 10px;
+  padding-top: 32px;
+  padding-left: 25px;
+  padding: 15px;

```

Comment:
we're having a little padding party here 🎉 
what do you think will happen here? Which padding will be applied?

Reply:
The last one 15px will be applied, sorry I didn't refactor the css code well